### PR TITLE
Update warm service class conformance graph labeling

### PR DESCRIPTION
**Pull request by BigLep**
- More clearly calling out the difference between the "warm service class conformance graphs"
- Updating the discussion to meeting SLOs rather than meeting SLIs.

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -121,13 +123,12 @@ Service Classes and their corresponding Service Level Objectives are defined in
...
   resize((width) => Plot.plot(addPlotCaption({
-    title: "\"Warm\" Service Class Conformance",
-    subtitle: html`Providers conforming to warm service class`,
+    title: "\"Warm\" Service Class Conformance: Retrievability",
+    subtitle: html`Providers meeting the "Warm" service class retrievability SLO`,
     x: { label: "Date" },
     y: {
       grid: true,
-      label: "Meet Retrieval SLI",
-      domain: [0, 100]
```
Comment (BigLep) [inline src/index.md:130]: I assume we don't want a fixed domain since this will grow based on the number of SPs who conform.

File: src/index.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ This dashboard is a scrappy work-in-progress to help spur discussion at [FDS-5 B
 const daily_metrics = FileAttachment("data/daily_metrics.csv").csv({typed: true});
 const daily_providers_metrics = FileAttachment("data/daily_providers_metrics.csv").csv({typed: true});
 
+// TODO: factor out these helper functions so they can be used on the "provider detail pages" as well.
```
Comment (BigLep) [inline src/index.md:16]: I should create an issue for this.  I'm just putting this here for now.

File: src/index.md (hunk 1 above)
Comment (davidgasquez) [reply src/index.md:130]: Yes! This was a leftover from when I started with percentages.

**Review by davidgasquez — APPROVED** [review]
Thanks for the PR and sorry I didn't go through a proper PR for my changes. Just pushed everything to main. :see_no_evil:

**Pull request merged**